### DISCUSSION TITLE
Logic/logic02/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#4] [General]
# Logic/logic02/

HobbitQia的笔记本

http://note.hobbitqia.cc/Logic/logic02/

**Comment by TonyCrane** [comment]
我超，酱紫细，kami/拜谢/拜谢/拜谢

**Comment by shadowash0215** [comment]
- $\overline{X \oplus Y} = X\overline{Y} + \overline{X}Y$: 
    - 应该是 $\overline{X \oplus Y} = XY + \overline{X}\overline{Y}$；
- 三变量异或的等式左侧并没有三个变量喵。

**Reply by The0xKa1** [reply]
- Maxterms 的求法不是很自然, 一般先求反函数的 Minterms 再将他们取反转为 Minterms.
   - Maxterms 的求法不是很自然, 一般先求反函数的 Minterms 再将他们取反转为 Maxterms.
